Source/Core/Core/IOS/USB/OH0/OH0.cpp

```diff
@@ -252,7 +252,7 @@ std::pair<ReturnCode, u64> OH0::DeviceOpen(const u16 vid, const u16 pid)
     has_device_with_vid_pid = true;
 
     if (m_opened_devices.find(device.second->GetId()) != m_opened_devices.cend() ||
-        !device.second->Attach(0))
+        !device.second->Attach())
```

Comment:
Since that `if` spans multiple lines, i think we should add some curlies in here...

Reply:
Good catch, fixed!